doc/INSTALL.rst

```diff
@@ -75,7 +75,7 @@ or:
     git clone https://github.com/assembl/assembl.git
     cd assembl
     fab -f assembl/fabfile.py -c assembl/configs/develop.rc install_single_server
-    fab -f assembl/fabfile.py -c assembl/configs/develop.rc bootstrap_from_checkout
+    VIRTUAL_ENV=$(pwd)/venv fab -f assembl/fabfile.py -c assembl/configs/develop.rc bootstrap_from_checkout
```

Comment:
why only changing that line and not the one above at the same time ?

Reply:
`VIRTUAL_ENV` is not needed for `bootstrap_from_checkout` fab task.